ebusd-2.1.x/en/vaillant/15.700.csv

```diff
@@ -304,5 +304,24 @@ r;w,,z2CoolingTimer.Thursday,timer cooling thursday,,,,03,,,slot1-3,,,
 r;w,,z2CoolingTimer.Friday,timer cooling friday,,,,04,,,slot1-3,,,
 r;w,,z2CoolingTimer.Saturday,timer cooling saturday,,,,05,,,slot1-3,,,
 r;w,,z2CoolingTimer.Sunday,timer cooling sunday,,,,06,,,slot1-3,,,
+# timers zone 3,,,,,,,,,,,,,
```

Comment:
please combine the two changes #369 and #370 into one

Reply:
Done